### ISSUE TITLE
[Bug] Wallet deamon not started

### ISSUE THREAD

**Issue by Delari** [Chia-Network/chia-blockchain#10663]
### What happened?

After installing Chia 1.3.0 some time later Node was restarted - at the current time it cannot start because waiting for the wallet daemon to start

when trying to start using powershell, it gives an error

### Version

1.3.0

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
PS C:\Users\user\AppData\Local\chia-blockchain\app-1.3.0\resources\app.asar.unpacked\daemon> .\chia.exe start wallet
Daemon not started yet
Starting daemon
(Unlock Keyring) Passphrase:
Unlocking daemon keyring
chia_wallet: started
PS C:\Users\user\AppData\Local\chia-blockchain\app-1.3.0\resources\app.asar.unpacked\daemon> Traceback (most recent call last):
  File "Lib\site-packages\chia\server\start_wallet.py", line 99, in <module>
  File "Lib\site-packages\chia\server\start_wallet.py", line 83, in main
KeyError: 'testing'
[364] Failed to execute script 'start_wallet' due to unhandled exception!
```

**Comment by wallentx** [comment]
Are you able to stop chia, delete the .chia\mainnet\config\config.yaml, and then restart chia?

**Comment by Delari** [comment]
> Are you able to stop chia, delete the .chia\mainnet\config\config.yaml, and then restart chia?

The problem was solved after installing update 1.3.1
